core/src/main/java/com/graphhopper/routing/util/BikeCommonFlagEncoder.java

```diff
@@ -679,7 +689,7 @@ public double getDouble( long flags, int key )
 
     boolean isPushingSection( OSMWay way )
     {
-        return way.hasTag("highway", pushingSections) || way.hasTag("railway", "platform");
+        return way.hasTag("highway", pushingSections) || way.hasTag("railway", "platform") || way.hasTag("bicycle", "dismount");
```

Comment:
this is now a bit confusing isPushingSection vs. `pushingSection` variable: maybe the variable should be renamed to `pushingSectionHighways`?